src/fides/api/ops/api/v1/endpoints/config_endpoints.py

```diff
@@ -33,7 +33,7 @@ def get_config(
     logger.info("Getting the exposable Fides configuration")
     if api_set:
         logger.info("Retrieving api-set application settings")
-        return ApplicationConfig.get_api_set_config(db)
+        return ApplicationConfig.get_api_set(db)
```

Comment:
Do we also need to censor sensitive config that was set in the API?

Reply:
updated in latest commits - please confirm!